src/promptflow/promptflow/_cli/_pf/entry.py

```diff
@@ -89,6 +95,16 @@ def entry(argv):
             init_finish_time - start_time,
             invoke_finish_time - init_finish_time,
         )
+        activity_name = f"pf.{args.action}s.{args.sub_action}.e2e"
+        message = f"{activity_name}.complete"
```

Comment:
shall we also add a start point? cc @D-W-

Reply:
Yes, need a start point

Reply:
ok